homeassistant/components/ezviz/const.py

```diff
@@ -10,6 +10,9 @@
 ATTR_AWAY = "AWAY_MODE"
 ATTR_TYPE_CLOUD = "EZVIZ_CLOUD_ACCOUNT"
 ATTR_TYPE_CAMERA = "CAMERA_ACCOUNT"
+CONF_SESSION_ID = "session_id"
+CONF_RFSESSION_ID = "rf_session_id"
```

Comment:
Please separate words with underscore in variable names, `CONF_RF_SESSION_ID`.

Reply:
Addressed in https://github.com/home-assistant/core/pull/94384